style.css

```diff
@@ -0,0 +1,823 @@
+}
+
+.btn-up {
+  margin-top: 22px;
+  font-weight: 700;
+}
+
+.btn span {
+  cursor: pointer;
+  position: relative;
+  transition: 0.5s;
+}
+
+.grid-item .project-role-wrapper span {
+  font-weight: bold;
+}
+
+.btn span::after {
+  content: "\00bb";
+  position: absolute;
+  opacity: 0;
+  top: -11px;
+  font-size: xx-large;
+  right: -20px;
+  transition: 0.5s;
+}
+
+.btn:hover {
+  color: #fff;
+  background: #6070ff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+}
+
+.btn:hover span {
+  padding-right: 25px;
+}
+
+.btn:hover span::after {
+  opacity: 1;
+  right: 0;
+}
+
+.btn:focus {
+  color: #fff;
+  background-color: #2230d2;
+}
+
+.btn.disabled {
+  color: #5e6c84;
+  border: 1px solid #5e6c84;
+}
+
+.about-me {
+  margin: 114px 0 0 0;
+  background-color: #fff;
+  padding-bottom: 214px;
+  border-top-right-radius: 120px;
+}
+
+.about-me ul {
+  list-style-type: none;
+}
+
+.about-me .menu {
+  margin: 0 36px;
+  width: calc(100% - 72px);
+}
+
+.about-me .menu li a {
+  color: #000;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.Modal .tags-wrapper ul {
+  margin-bottom: 10px;
+  border-bottom: 1px solid #e5e5e5;
+}
+
+.about-me .menu li ul {
+  display: flex;
+  margin: 36px 0;
+  flex-direction: column;
+}
+
+.about-me .menu li ul li a {
+  color: #253858;
+  display: flex;
+  padding: 10px 20px;
+  margin-bottom: 12px;
+  border-radius: 8px;
+  align-items: center;
+  justify-content: flex-start;
+  background-color: #f7f7f9;
+}
+
+.about-me .menu .closed {
+  margin: 24px 0;
+  font-weight: 500;
+  padding-bottom: 23px;
+  border-bottom: 1px solid #dfe1e6;
+  background: url("chevron-right.svg") no-repeat calc(100% - 64px) center;
+}
+
+.about-me .menu .open {
+  background: url("carret.svg") no-repeat calc(100% - 64px) 0;
+}
+
+.Modal .btn {
+  width: 140px;
+  font-size: 14px;
+}
+
+.Modal .btn img {
+  margin: 0 0 0 3px;
+}
+
+.about-me .menu li ul li a img {
+  margin-right: 16px;
+}
+
+.grid-item .card-image img:hover {
+  cursor: pointer;
+}
+
+.menu-wrapper .menu .mobile-menu {
+  position: fixed;
+  left: 0;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  height: 100vh;
+  z-index: 1;
+  color: #fff;
+  display: flex;
+  font-size: 32px;
+  line-height: 44px;
+  font-weight: 600;
+  flex-direction: column;
+  background-color: rgba(50, 70, 255, 0.8);
+  backdrop-filter: blur(5.5px);
+}
+
+.menu-wrapper .menu .mobile-menu li:first-child {
+  margin: 49px 38px 0;
+}
+
+.menu-wrapper .menu .mobile-menu li a {
+  color: #fff;
+  font-weight: 600;
+}
+
+.Modal {
+  position: fixed;
+  display: none;
+  height: 100vh;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  top: 0;
+  z-index: 3;
+  justify-content: center;
+  align-items: center;
+  background-color: rgba(193, 199, 208, 0.75);
+  backdrop-filter: blur(5px);
+  padding: 10px 20px;
+}
+
+.Modal .grid-item {
+  margin: 10px 0;
+  height: 90vh;
+  overflow-y: scroll;
+}
+
+.Modal .grid-item .btn,
+.Modal .card-image {
+  margin-top: 20px;
+}
```

Comment:
- [ ] On desktop devices, the height of the image is not big enough to see what it's about. It also doesn't look like the Figma design. Please make changes 🙏 
- [ ] According to the design, the 'See Live' and 'See Source' buttons should also be on the same line rather than stacked on top of each other.
### Your Design
![image](https://user-images.githubusercontent.com/46839548/168442452-6f8cf483-2107-41d4-8b14-cbe68fd389ef.png)
### Figma Design
![image](https://user-images.githubusercontent.com/46839548/168442503-a0e758e9-af92-4dc6-92d3-0885e9dc5bf9.png)